lib/screen/feed/feed_preview_page.dart

```diff
@@ -114,7 +116,6 @@ class _FeedPreviewScreenState extends State<FeedPreviewScreen>
     routeObserver.unsubscribe(this);
     WidgetsBinding.instance.removeObserver(this);
     Sentry.getSpan()?.finish(status: const SpanStatus.ok());
-    // _controller.dispose();
```

Comment:
why remove this?

Reply:
we called dispose() in editorial page, together with another controller

Reply:
ahh okay, it's just a commented code.